src/main/fc/fc_init.c

```diff
@@ -478,8 +478,8 @@ void init(void)
     initBoardAlignment(boardAlignment());
 
     if (!sensorsAutodetect()) {
-        // if gyro was not detected due to whatever reason, we give up now.
-        failureMode(FAILURE_MISSING_ACC);
+        // if gyro was not detected due to whatever reason, don't arm.
+        setArmingDisabled(ARMING_DISABLED_NO_GYRO);
```

Comment:
This failure mode triggered a visual indicator... we should probably include a visual indicator using LEDs